src/components/biggive-tipping-slider/biggive-tipping-slider.tsx

```diff
@@ -20,7 +20,10 @@ export class BiggiveTippingSlider {
 
   @Prop() donationAmount: number;
 
-  @Prop() donationCurrency: '£';
+  /**
+   * ISO-4217 currency code (e.g. GBP, USD)
+   */
+  @Prop() donationCurrency!: string;
```

Comment:
Nice docblock. Maybe worth using `'GBP'` as the type instead of `string`? Or something like `'GBP'|'USD'`. Ideally it would be only the currencies that the slider is intended to support, which afaik is only GBP right now but we want to be able to expand it easily in future.